countries.html

```diff
@@ -0,0 +1,34 @@
+<html>
+
+<head>
+    <title>Countries</title>
+    <link rel="stylesheet" href="countries.css" />
+</head>
+
+<body>
+
+    <div class="main-container">
+        <p class="quizHeading">Countries</p>
+        <div class="scoreboard">
+            <div class="question-number">
+                Question: 1/5
+            </div>
+            <div class="score">
+                Score: 0
+            </div>
+        </div>
+        <div class="question-container">
+            <div class="question">
+                Which country has the largest muslim population?
+            </div>
+            <div class="answer-options">
+                <div class="answer-option option1">Iraq</div>
+                <div class="answer-option option2">Nigeria</div>
+                <div class="answer-option option3">Indonesia</div>
+                <div class="answer-option option4">Libya</div>
+            </div>
+        </div>

```

Comment:
Try to use semantic HTML, which in this case would be to use a form with radio buttons to choose answers. You could then customise the radio buttons. But this works as well, just a matter of preference.